subtests/docker_cli/events/test_event_parsing.py

```diff
@@ -0,0 +1,167 @@
+                                      'identifier': fqins[i],
+                                      'operation':  actions[i],
+                                      'source':     None}))
+        actual = events.parse_events(event_log)
+        self.assertEqual(actual, expect)
+
+    def test_1_10(self):
+        """
+        docker-1.10 format: container and network events
+        """
+        t = ["2016-04-06T09:54:06.640353011-04:00",
+             "2016-04-06T09:54:07.096096286-04:00",
+             "2016-04-06T09:54:07.108386649-04:00",
+             "2016-04-06T09:54:07.442663135-04:00",
+             "2016-04-06T09:54:07.846336619-04:00",
+             "2016-04-06T09:55:00.000000000-04:00"]
+        cid = ("ae5f8ee3cdc14512fbc6f908f06cc859"
+               "0b53d0eb9e36cf543165d130f1169a0e")
+
+        nid = ("aa5e8221edf314708c2e0fbc80b671b2"
+               "e11464c87c5bd52b93a01c38538d978b")
+
+        source = "registry.access.redhat.com/rhel7/rhel:latest"
+
+        event_log = ("{t[0]} container create {cid}"
+                     " (Release=56, Vendor=Red Hat, Inc., Name=rhel7/rhel,"
+                     " Authoritative_Registry=registry.access.redhat.com,"
+                     " BZComponent=rhel-server-docker,"
+                     " Build_Host=rcm-img03.build.eng.bos.redhat.com,"
+                     " Version=7.2,"
+                     " image=registry.access.redhat.com/rhel7/rhel:latest,"
+                     " name=events_c36O, Architecture=x86_64)"
+                     "\n"
+                     "{t[1]} network connect {nid}"
+                     " (container={cid}, name=bridge, type=bridge)"
+                     "\n"
+                     "{t[2]} container start {cid}"
+                     " (Build_Host=somewhere.redhat.com,"
+                     " Name=rhel7/rhel, Vendor=Red Hat, Inc., Version=7.2,"
+                     " image=registry.access.redhat.com/rhel7/rhel:latest,"
+                     " name=events_c36O, Architecture=x86_64,"
+                     " Authoritative_Registry=registry.access.redhat.com,"
+                     " BZComponent=rhel-server-docker, Release=56)"
+                     "\n"
+                     "{t[3]} container die {cid}"
+                     " (BZComponent=rhel-server-docker, Release=56,"
+                     " Vendor=Red Hat, Inc.,"
+                     " image=registry.access.redhat.com/rhel7/rhel:latest,"
+                     " Version=7.2, name=events_c36O, Architecture=x86_64,"
+                     " Authoritative_Registry=registry.access.redhat.com,"
+                     " Build_Host=somewhere.redhat.com,"
+                     " Name=rhel7/rhel)"
+                     "\n"
+                     "{t[4]} network disconnect {nid}"
+                     " (name=bridge, type=bridge, container={cid})\n"
+                     "{t[5]} container archive-path {cid}"
+                     " (ignore this)\n"
+                     "".format(**locals()))
```

Comment:
A separate file wouldn't work well with the {cid} {etc} refactoring. Triple-quotes don't work because the lines are too long, triggering E501. But I've made a commit (unpushed) that cleans this up somewhat.